### PR TITLE
Resolved the `framework_upgrade` CI test failure

### PR DESCRIPTION
**Pull request by junkil-park**
### Description

- Provide a valid URL for discussion, which resolves the framework_upgrade CI test failure
- Bump up the base commit to 1.9.5 for the compat test and framework_upgrade test

### CODE REVIEW

**Review by thepomeranian — APPROVED** [review]

File: aptos-move/aptos-release-builder/data/release.yaml [hunk 1]
```diff
@@ -33,9 +33,9 @@ proposals:
             - refundable_bytes
   - name: step_4_enable_fairness_shuffler
     metadata:
-      title: "Enable Fairness Shuffler"
-      description: "FairnessShuffler that tries to spread out (delay) transactions from dominant modules in a block, so that if block limit is hit, transactions from non-dominant modules have a fair share of the block space."
-      discussion_url: "TBA"
+      title: "AIP-68: Enable Fairness Shuffler"
```
[the remark this answers is not in the document]
Comment (msmouse) [reply aptos-move/aptos-release-builder/data/release.yaml]: I added a commit

File: .github/workflows/docker-build-test.yaml [hunk 2]
```diff
@@ -329,7 +329,7 @@ jobs:
     with:
       GIT_SHA: ${{ needs.determine-docker-build-metadata.outputs.gitSha }}
       FORGE_TEST_SUITE: framework_upgrade
-      IMAGE_TAG: aptos-node-v1.8.3 # This workflow will test the upgradability from the current tip of the release branch to the current main branch.
+      IMAGE_TAG: testnet # This workflow will test the upgradability from the current tip of the release branch to the current main branch.
```
[the remark this answers is not in the document]
Comment (perryjrandall) [reply .github/workflows/docker-build-test.yaml:332]: you should explicilt change this ot the current version (1.9.5?)

[the remark this answers is not in the document]
Comment (junkil-park) [reply .github/workflows/docker-build-test.yaml:332]: @perryjrandall , then, this line should be changed too, right?
https://github.com/aptos-labs/aptos-core/pull/12046/files#diff-8b02be56513144a070e205ed7fcd61b9162d665d5f451afbe80884768b62fe87R305

[the remark this answers is not in the document]
Comment (junkil-park) [reply .github/workflows/docker-build-test.yaml:332]: Just to double check, `framework_upgrade` should be done against `aptos-framework-v1.9.0` or `aptos-node-v1.9.5`

File: .github/workflows/docker-build-test.yaml [hunk 3]
```diff
@@ -302,7 +302,7 @@ jobs:
     with:
       GIT_SHA: ${{ needs.determine-docker-build-metadata.outputs.gitSha }}
       FORGE_TEST_SUITE: compat
-      IMAGE_TAG: testnet # test against a previous testnet release
```
Comment (igor-aptos) [inline .github/workflows/docker-build-test.yaml:305]: let's leave testnet

File: .github/workflows/docker-build-test.yaml [hunk 4]
```diff
@@ -329,7 +329,7 @@ jobs:
     with:
       GIT_SHA: ${{ needs.determine-docker-build-metadata.outputs.gitSha }}
       FORGE_TEST_SUITE: framework_upgrade
-      IMAGE_TAG: aptos-node-v1.8.3 # This workflow will test the upgradability from the current tip of the release branch to the current main branch.
```
Comment (igor-aptos) [inline .github/workflows/docker-build-test.yaml:332]: should this also be testnet?

File: .github/workflows/docker-build-test.yaml (hunk 3 above)
Comment (junkil-park) [reply .github/workflows/docker-build-test.yaml:305]: @perryjrandall

File: .github/workflows/docker-build-test.yaml (hunk 4 above)
Comment (junkil-park) [reply .github/workflows/docker-build-test.yaml:332]: @perryjrandall ?

File: .github/workflows/docker-build-test.yaml (hunk 3 above)
Comment (igor-aptos) [reply .github/workflows/docker-build-test.yaml:305]: Actually that will not work when we update testnet with 1.10
 This needs to be 1.9.5...

**Review by sherry-x — APPROVED** [review]

**Review by igor-aptos — APPROVED** [review]

**Pull request merged**